.env-devel

```diff
@@ -154,6 +154,7 @@ SMTP_PASSWORD=it_doesnt_matter
 SMTP_PROTOCOL=UNENCRYPTED
 
 SIMCORE_SERVICES_NETWORK_NAME=interactive_services_subnet
+SIMCORE_INTERACTIVE_NETWORK_SUBNET=172.16.0.0/16
```

Comment:
I hope this will not immediately intersect with some local default networks created by docker

Reply:
you are right I have change it so there is no collision